pkg/config/config_evaluator.go

```diff
@@ -36,14 +36,14 @@ const (
 type GitHubClientProvider func(ctx context.Context, org, repo string) (*github.Client, error)
 
 type ConfigEvaluator interface {
-	Eval(ctx context.Context, org, repo, scope string, token interface{}) (*Scope, error)
+	Eval(ctx context.Context, org, repo, scope string, token interface{}) (*Scope, string, error)
```

Comment:
Perhaps drop a comment here explaining what the returned string is

Reply:
done